db/migrate/20170905111153_create_user_agents.rb

```diff
@@ -0,0 +1,7 @@
+class CreateUserAgents < ActiveRecord::Migration
+  def change
+    create_table :user_agents do |t|
+      t.string :user_agent_string,           :limit => 1000
```

Comment:
Following on from the comment on the `AddIndexToUserAgent` migration, we can add the index here directly with the `index: true` option to the column definition.  I think it'll be useful to mark the column as not nullable by passing `null: false` as well.

Reply:
http://edgeguides.rubyonrails.org/active_record_migrations.html#column-modifiers is a useful resource for migrations and the specific options you can pass when creating columns.  (The whole guide is useful, but that section in particular is useful here).